tools/test.py

```diff
@@ -1363,71 +1363,127 @@ def ReadConfigurationInto(path, sections, defs):
-      default=[], action="append")
-  result.add_option("-t", "--timeout", help="Timeout in seconds",
-      default=120, type="int")
-  result.add_option("--arch", help='The architecture to run tests for',
-      default='none')
-  result.add_option("--snapshot", help="Run the tests with snapshot turned on",
-      default=False, action="store_true")
-  result.add_option("--special-command", default=None)
-  result.add_option("--node-args", dest="node_args", help="Args to pass through to Node",
-      default=[], action="append")
-  result.add_option("--expect-fail", dest="expect_fail",
-      help="Expect test cases to fail", default=False, action="store_true")
-  result.add_option("--valgrind", help="Run tests through valgrind",
-      default=False, action="store_true")
-  result.add_option("--check-deopts", help="Check tests for permanent deoptimizations",
-      default=False, action="store_true")
-  result.add_option("--cat", help="Print the source of the tests",
-      default=False, action="store_true")
-  result.add_option("--flaky-tests",
-      help="Regard tests marked as flaky (run|skip|dontcare)",
-      default="run")
-  result.add_option("--warn-unused", help="Report unused rules",
-      default=False, action="store_true")
-  result.add_option("-j", help="The number of parallel tasks to run",
-      default=1, type="int")
-  result.add_option("-J", help="Run tasks in parallel on all cores",
-      default=False, action="store_true")
-  result.add_option("--time", help="Print timing information after running",
-      default=False, action="store_true")
-  result.add_option("--suppress-dialogs", help="Suppress Windows dialogs for crashing tests",
-        dest="suppress_dialogs", default=True, action="store_true")
-  result.add_option("--no-suppress-dialogs", help="Display Windows dialogs for crashing tests",
-        dest="suppress_dialogs", action="store_false")
-  result.add_option("--shell", help="Path to V8 shell", default="shell")
-  result.add_option("--store-unexpected-output",
-      help="Store the temporary JS files from tests that fails",
-      dest="store_unexpected_output", default=True, action="store_true")
-  result.add_option("--no-store-unexpected-output",
-      help="Deletes the temporary JS files from tests that fails",
-      dest="store_unexpected_output", action="store_false")
-  result.add_option("-r", "--run",
-      help="Divide the tests in m groups (interleaved) and run tests from group n (--run=n,m with n < m)",
-      default="")
-  result.add_option('--temp-dir',
-      help='Optional path to change directory used for tests', default=False)
-  result.add_option('--repeat',
-      help='Number of times to repeat given tests',
-      default=1, type="int")
-  result.add_option('--abort-on-timeout',
-      help='Send SIGABRT instead of SIGTERM to kill processes that time out',
-      default=False, action="store_true", dest="abort_on_timeout")
-  return result
+    result = optparse.OptionParser()
```

Comment:
The line above (` def BuildOptions(): `) has 0 indent, this line should have 2 right (as the whole file has 2-space indent, except for wrapped lines etc.)?

Reply:
The top level functions in test.py (ex. `def ProcessOptions(options):` , `def ReadConfigurationInto(path, sections, defs):` , `def SplitPath(s):`, etc.) all are on 0 indent level. I kept the `def BuildOptions():` as is to maintain that formatting, hence no change on the line above.

The lines inside the function `def BuildOptions():` (line # 1366-1486 here) - are on 4 indent level for first indent as per the pep8 recommendation.

Reply:
PEP-8 recommends using the indentation that the rest of the project uses for normal indentation, which in Node core is 2-space.